packages/middleware/src/types/fileUpload.ts

```diff
@@ -0,0 +1,7 @@
+export interface FileUploadOptions {
+  enabled?: boolean;
```

Comment:
hmm just thinking out loud - wouldn't be easier to remove `enabled` (whenever the file upload config is provided, file upload is enabled) flag or set it to true `enabled: true`?

Reply:
@bartoszherba I meant that we could set the type to accept only `true` value

```ts
export interface FileUploadOptions {
  enabled: true;
```

so there is no way to set `enable: false` and provide other configuration options